app/serializers/survey_list_serializer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class SurveyListSerializer < ApplicationSerializer
```

Comment:
Minor preference for:

```suggestion
        render json: SurveySimpleSerializer.new(surveys, meta: meta_from_pagy(pagy))
```

As used in previous projects: https://github.com/nimblehq/the1-api/tree/development/engines/the1_reward/app/serializers/the1_reward/v2

Reply:
Thanks! Fixed in https://github.com/nimblehq/nimble-survey-web/pull/71/commits/cdd4a1cef1a2ffde5f0171f2c210a9d32694b413

Reply:
@olivierobert Please help me check it again 🙇